common/src/main/java/earth/terrarium/ad_astra/common/util/OxygenUtils.java

```diff
@@ -136,8 +136,10 @@ public static void deoxygenizeBlocks(ServerLevel level, Set<BlockPos> entries, B
                 }
 
                 if (block instanceof TorchBlock && !block.equals(Blocks.SOUL_TORCH) && !block.equals(Blocks.SOUL_WALL_TORCH)) {
-                    level.setBlockAndUpdate(pos, ModBlocks.EXTINGUISHED_TORCH.get().defaultBlockState());
-                    continue;
+                    if (!block.equals(Blocks.REDSTONE_TORCH) && !block.equals(Blocks.REDSTONE_WALL_TORCH)) {
```

Comment:
This should  likely be changed to a tag check as to make better compatibility with other mods

Reply:
Then to make it perfect, i think it would be a good idea to add a deoxygenation recipe type.
Since we don't know exactly what the items added in other mods are made of,
It should be specified in the recipe json.

Reply:
Ok yeah  a recipe json would be best as for now ill approve this so it doesn't break redstone torches in the mean time.